Changed 'ListingCanceled' to 'ListingCancelled' in contract , Added directory for graph codegen , Changed the directory for the subgraph .git folder removal

Dhruv-2003 opened the pull request:
Made 3 changes : 

1 . Changed 'ListingCanceled' to 'ListingCancelled' in contract , as in the graphQL query
2.  Added directory for graph codegen 
3.  Changed the 'cd hardhat' to 'cd subgraph'  for .git file removal in the subgraph directory

haardikk21 commented:
Ah, `ListingCanceled` vs `ListingCancelled`. We've gone back and forth on this before, I want to stick to `Canceled` (single 'L') now - it's really just an American vs British spelling thing. Let's modify the Graph query instead of changing it across the contract stuff because it looks like the graph query is a leftover remnant because other things were changed before.

rest looks good to me.

haardikk21 commented:
Wait, it also looks like you edited the wrong file for subgraph stuff. You made changes in Part-3 which is the website tutorial, and replaced it with the subgraph tutorial. Those changes should've been in Part 2.

Dhruv-2003 commented:
> Wait, it also looks like you edited the wrong file for subgraph stuff. You made changes in Part-3 which is the website tutorial, and replaced it with the subgraph tutorial. Those changes should've been in Part 2.

Sorry for that mistake , I have updated the page

Dhruv-2003 commented:
> Ah, `ListingCanceled` vs `ListingCancelled`. We've gone back and forth on this before, I want to stick to `Canceled` (single 'L') now - it's really just an American vs British spelling thing. Let's modify the Graph query instead of changing it across the contract stuff because it looks like the graph query is a leftover remnant because other things were changed before.
> 
> rest looks good to me.

I have modified the graph query now keeping the contract as same with canceled. So it is canceled all across . Moreover other changes are not yet reflecting on the website , it is still the old one .

Dhruv-2003 commented:
will this PR be merged ?? The issue is still present on the website

haardikk21 commented on NFT-Marketplace-Part-2.md:
```diff
@@ -282,7 +282,7 @@ type ListingEntity @entity {
 Great, we're now finally at the point where we can write our actual script.
 
-One last quick thing before we do that though, run the following command in your terminal
+One last quick thing before we do that though, run the following command in your terminal from the `supgrah` directory
```
Can we fix the typo here - `subgraph` instead of `supgrah`

Dhruv-2003 replied:
typo fixed
